Makefile

```diff
@@ -20,8 +30,19 @@ run:
 		-p 27015:27015/udp \
 		-p 27020:27020/udp \
 		-p 27020:27020/tcp \
-		-e "SERVER_HOSTNAME=test" \
-		-e "SERVER_PASSWORD=test" \
-		-e "RCON_PASSWORD=test" \
+		-e "SERVER_HOSTNAME=$(SERVER_HOSTNAME)" \
+		-e "SERVER_PASSWORD=$(SERVER_PASSWORD)" \
+		-e "RCON_PASSWORD=$(RCON_PASSWORD)" \
 		-e "STEAM_ACCOUNT=$(STEAM_ACCOUNT)" \
-		$(IMAGE_NAME)
\ No newline at end of file
+		-e "TICKRATE=$(TICKRATE)" \
+		-e "GAME_TYPE=$(GAME_TYPE)" \
+		-e "GAME_MODE=$(GAME_MODE)" \
+		-e "MAP=$(MAP)" \
+		-e "MAPGROUP=$(MAPGROUP)" \
+		-e "MAXPLAYERS=$(MAXPLAYERS)" \
+		--name $(CONTAINER_NAME)" \
```

Comment:
it looks like an extra quote at the end of this line that needs to be removed. it's causing this error:

```bash
/bin/bash: -c: line 0: unexpected EOF while looking for matching `"'
/bin/bash: -c: line 1: syntax error: unexpected end of file
make: *** [run] Error 2
```